### ISSUE TITLE
Allow pytest to be executed simultaneously

### ISSUE THREAD

**Issue by maringuu** [fkie-cad/FACT_core#948]
Since the tests take quite a bit to execute, especially integration and acceptance tests it would be great to speedup by parallel execution.

There are several blockers to for this.
The one I remember is database access. Tables are modified, dropped etc by fixtures and tests and this obviously causes errors.

**Comment by jstucke** [comment]
I don't think this could work for the integration and acceptance tests when stuff is written to the DB, Redis, etc. because this could lead to side effects (unless there are some workarounds like e.g. using multiple test DBs but it's probably not worth the effort). 

The unit tests on the other hand should work (at least in theory). I don't know if it's worth the effort here as well when the unit tests need about 45 seconds to run without parallelism.
